docs/sphinx/source/whatsnew/v0.10.0.rst

```diff
@@ -17,7 +17,9 @@ Breaking changes
 
 Deprecations
 ~~~~~~~~~~~~
-
+* :func:`~pvlib.iotools.get_pvgis_tmy` and :func:`~pvlib.iotools.read_pvgis_tmy`
+  now rename columns to standard pvlib names by default (``map_variables=True``)
+  (:pull:`1772`)
```

Comment:
Should this go in `Breaking changes`?  I think of `Deprecations` as being for newly deprecated things, not things that reached the end of their deprecation period.